docs/Development Concepts/Protocols/2.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Protocol Identifiers
```

Comment:
The protocol id should not matter to the user as it is set by the Waku library. I'd focus this section on an overview of the protocols instead.